spec/models/eligibility_check_spec.rb

```diff
@@ -98,12 +98,14 @@
       before { eligibility_check.country_code = "IN" }
 
       context "when teach_children is false" do
+        before { create(:country, :subject_limited, code: "IN") }
         before { eligibility_check.teach_children = false }
 
         it { is_expected.to include(:teach_children_secondary) }
       end
 
       context "when qualified_for_subject is false" do
+        before { create(:country, :subject_limited, code: "IN") }
```

Comment:
To avoid duplication, this line can go in the `"when filtering by subject"` context, and then it'll apply to both the `"when teach_children is false"` and `"when qualified_for_subject is false"` context.